Restructure edx models code

quitterie-lcs opened the pull request:
## Purpose

Based on the structure of xapi models implementation, the structure of edx models for already implemented events has been improved.

## Proposal

- [x] navigational events models
- [x] enrollment events models

SergioSim commented on tests/models/edx/test_navigational.py:
```diff
@@ -94,11 +123,33 @@ def test_models_edx_ui_seq_goto_selector_with_valid_event(event):
-    assert ModelSelector(module="ralph.models.edx").get_model(event) is UISeqNext
+    assert statement.event_type == "seq_next"
+    assert statement.name == "seq_next"
+    assert statement.event.new == statement.event.old + 1
```
```suggestion
```
As these values are preset in the test we don't need to test them ?

SergioSim commented on tests/models/edx/test_navigational.py:
```diff
@@ -140,11 +191,33 @@ def test_models_edx_ui_seq_next_event_with_invalid_content(old, new, event):
+
+    assert statement.event_type == "seq_prev"
+    assert statement.name == "seq_prev"
+    assert statement.event.new == statement.event.old - 1
```
```suggestion
```
here too ?

SergioSim reviewed (approved):
LGTM) I like this new structure)
Have only a couple of suggestions and one minor concern - now in the edx module we use `event` and `statement` interchangeably which might be a bit confusing.
(ex. `edx.enrollment.statements.EdxCourseEnrollmentActivated(BaseServerEvent)`)
But I guess as the meaning is similar this should be OK)
![image](https://media.giphy.com/media/a5llhdWKMOIfneMCpB/giphy.gif)

quitterie-lcs commented:
@SergioSim Thanks for your review. I have updated with your proposition. 
About the `statement/server` issue, I cleaned up as much as I could or when I thought it was relevant to use `statement` rather than `event` (that is what we did for `xapi` ones). In the last commit, I have removed `event` suffix in model naming and just let the name of the model or with `model` suffix for `base` models (always to be coherent with `xapi` modelization). WDYTOI?

SergioSim commented:
> @SergioSim Thanks for your review. I have updated with your proposition.
> About the `statement/server` issue, I cleaned up as much as I could or when I thought it was relevant to use `statement` rather than `event` (that is what we did for `xapi` ones). In the last commit, I have removed `event` suffix in model naming and just let the name of the model or with `model` suffix for `base` models (always to be coherent with `xapi` modelization). WDYTOI?

@quitterie-lcs This makes it more consistent, thank you for going the extra mile !
